src/NativeScript/inspector/DomainInspectorAgent.cpp

```diff
@@ -0,0 +1,21 @@
+#include <JavaScriptCore/JSObject.h>
+#include "DomainInspectorAgent.h"
+#include "GlobalObjectInspectorController.h"
+
+namespace NativeScript {
+DomainInspectorAgent::DomainInspectorAgent(WTF::String domainName, GlobalObject& globalObject, JSC::JSCell* constructorFunction)
```

Comment:
Just a suggestion - you could put the `globalObject` parameter first to be in line with the rest of our classes.